internal/cmd/asyncapi/command_export.go

```diff
@@ -322,31 +321,22 @@ func (c *command) getBindings(clusterId, topicName string) (*bindings, error) {
 	customConfigMap := make(map[string]string)
 	topicConfigMap := make(map[string]any)
 
-	// Determine whether the given config value can be put into the AsyncAPI Kakfa bindings or put into our custom struct for extra configs
-	topicConfigOptions := types.NewSet(
-		"cleanup.policy",
-		"delete.retention.ms",
-		"max.message.bytes",
-		"retention.bytes",
-		"retention.ms",
-	)
+	// Determine whether the given config value can be put into the AsyncAPI Kafka bindings or put into our custom struct for extra configs
 	for _, config := range configs.Data {
-		if topicConfigOptions.Contains(config.GetName()) {
-			switch config.GetName() {
-			case "cleanup.policy":
-				topicConfigMap[config.GetName()] = strings.Split(config.GetValue(), ",")
-			case "max.message.bytes":
-				topicConfigMap[config.GetName()], err = strconv.ParseInt(config.GetValue(), 10, 32)
-				if err != nil {
-					return nil, err
-				}
-			default:
-				topicConfigMap[config.GetName()], err = strconv.ParseInt(config.GetValue(), 10, 64)
-				if err != nil {
-					return nil, err
-				}
+		switch config.GetName() {
+		case "cleanup.policy":
+			topicConfigMap[config.GetName()] = strings.Split(config.GetValue(), ",")
+		case "max.message.bytes":
+			topicConfigMap[config.GetName()], err = strconv.ParseInt(config.GetValue(), 10, 32)
+			if err != nil {
+				return nil, err
+			}
+		case "delete.retention.ms", "retention.bytes", "retention.ms":
```

Comment:
Thanks for making this change! I think `switch` uses a HashSet under the hood, so the runtime should be the same :smile: